.github/workflows/python_tests.yml

```diff
@@ -43,6 +43,10 @@ jobs:
         with:
           name: python-wheels
           path: wheels/
+      - uses: actions/upload-artifact@v3
+        with:
+          name: requirements
+          path: python/requirements.txt
```

Comment:
Where is the source of truth for this file? Where does the data come from? A comment would be helpful.

Reply:
What do you mean by that? The file is just being pulled out of the repository.

Reply:
Oh I see. I thought maybe it was coming from a genrule().

Does it need to be uploaded at all then? Could we use a regular checkout action to obtain it?

Reply:
Makes sense. A comment to that effect would be helpful. Otherwise I would assume that any artifact you are uploading was produced by the build process. It would help to know the rationale for why a source was being uploaded as an artifact.

Reply:
Makes sense, added a comment!